2016/proposals/singh-shubham.md

```diff
@@ -0,0 +1,210 @@
+# Manage workflow for instructor training
+
+## Abstract
+
+Computing Skills have now become essential more so in the field of Science, but many a time we find researchers struggling with code or tools to get things done.
+Software Carpentary is a step in this direction.
+Since 1998, Software Carpentry has been teaching researchers in science, engineering, medicine, and related disciplines the computing skills they need to get more done in less time and with less pain.
+Software Carpentary , uses AMY ( a django based web application) to to keep track of who's qualified to teach, what workshops have run, who taught and attended. As more and more people are applying to become instructors at Software Carpentary. It has become essential to manage the workflow for instructor training.
+
+This project aims at designing such a workflow and implementing it by adding appropriate views to AMY.
+
+## Proposed workflow
+
+![HTML MOCKUP flowchart](https://cloud.githubusercontent.com/assets/15982349/13770094/aa23d670-eaaa-11e5-81a8-312f860ecd7b.png)
+
+1. Interested Trainees apply as Group or Individuals by filling forms.
+![HTML MOCKUP FORMS](https://cloud.githubusercontent.com/assets/15982349/13816694/6c7a68e6-ebb5-11e5-809e-eb5bd00d7a22.png)
+2. All applications are listed in the *ALL_APPLICATIONS* view. In this view the lead trainer can accept or reject an application.
+The lead trainer has to compose one sample email by filling a web-form before accepting an application, this email will contain information such as
+date, location, mode(online or in person), instructor name and instructor email id. This information is also stored in the database for the accepted applicant to be queried on later.
+Accepting an application will send an email, to the applicant that he/she has been accepted into the program with other details about the actual training along with one randomly generated password to login in to their account (this password can be changed to anything by the trainee after the first login).
```

Comment:
Every member gets a login, because as I see it, even though a person is a part of the group every trainee is assessed individually.